Allow passing min/maxYear to combodate

dolzenko opened the pull request:
Currently maxYear is hardcoded to 2015, making the control unusable for us. Not sure what the best option is. This pull request adds pass through of `e-min/max-year` attributes to combodate control.

eugef commented:
@dolzenko, could you also update documentation to include new attributes

dolzenko commented:
@eugef I can try, you mean to https://vitalets.github.io/angular-xeditable? Should I also add the rest of the combodate options from https://vitalets.github.io/combodate/#docs?

eugef commented:
Just modify https://github.com/vitalets/angular-xeditable/blob/master/docs/demos/combodate to include new attributes and add reference to the https://github.com/vitalets/angular-xeditable/blob/master/docs/js/structure.js

dolzenko commented:
@eugef I have updated to allow passing all the options and added short doc, not sure about https://github.com/vitalets/angular-xeditable/blob/master/docs/js/structure.js :(. Also wasn't able to setup test runner, sorry.

lalitkapoor commented:
@eugef any help to get the test runner running? I'm running into a problem with that too!
